ansible/roles/species-list/files/v4.4.0_update.sql

```diff
@@ -0,0 +1,22 @@
+#Create a matched_species table
+    CREATE TABLE `matched_species` (
+           `id` int NOT NULL AUTO_INCREMENT,
+           `taxon_concept_id` varchar(255) DEFAULT NULL,
+           `scientific_name` varchar(255) NOT NULL,
+           `scientific_name_authorship` varchar(255) DEFAULT NULL,
+           `vernacular_name` varchar(255) DEFAULT NULL,
+           `kingdom` varchar(255) DEFAULT NULL,
+           `phylum` varchar(255) DEFAULT NULL,
+           `taxon_class` varchar(255) DEFAULT NULL,
+           `taxon_order` varchar(255) DEFAULT NULL,
+           `family` varchar(255) DEFAULT NULL,
+           `genus` varchar(255) DEFAULT NULL,
+           `taxon_rank` varchar(255) DEFAULT NULL,
+           `version` BIGINT NOT NULL DEFAULT 0,
+           PRIMARY KEY (`id`,`scientific_name`),
```

Comment:
Does the primary key need to be composite? it seems redundant as id is auto incrementing

Reply:
It is for indexing the scientific_name.  I quickly scan the docs of Mysql.  It says creating an index via adding it to primary key.  Correct me if I am wrong

Reply:
The way composite keys work a query for just the scientific name will not use the index, it will have to do a full table scan. Scientific name can be added as a 2nd index